hc/settings.py

```diff
@@ -112,7 +112,7 @@
         }
     }
 
-DATABASES['default'] = dj_database_url.config(default="postgres://postgres:@localhost:/hc")
+DATABASES['default'] = dj_database_url.config(default="postgres://postgres:1234567890@localhost:/hc")
```

Comment:
Why are we hardcoding the database? these settings should be picked from an environment variable